vnext/Desktop.ABITests/React.Windows.Desktop.ABITests.vcxproj

```diff
@@ -103,6 +103,9 @@
       <CppWinRTDynamicProjectWinMDReferences Include="$(OutputPath)..\React.Windows.Desktop\facebook.react.winmd">
         <WinMDPath>$(OutputPath)..\React.Windows.Desktop\facebook.react.winmd</WinMDPath>
       </CppWinRTDynamicProjectWinMDReferences>
+      <CppWinRTDynamicProjectWinMDReferences Include="$(OutputPath)..\React.Windows.Desktop\Microsoft.ReactNative.winmd">
```

Comment:
>CppWinRTDynamicProjectWinMDReferences [](start = 7, length = 37)

We also need to have a build dependency somewhere. The Microsoft.ReactNative.winmd must be created before the test project.

Reply:
We should get this indirectly through the dependency on the DLL in line 88.

---
In reply to: [423290781](https://github.com/microsoft/react-native-windows/pull/4848#discussion_r423290781) [](ancestors = 423290781)